src/main.js

```diff
@@ -59,6 +61,30 @@ let load = base.loader({
     },
   },
 
+  WorkerState: {
+    requires: ['cfg'],
+    setup: async ({cfg}) => {
+      let WorkerState = workerState.setup({
+        account: cfg.azure.account,
+        table: cfg.app.workerStateTableName,
+        credentials: cfg.taskcluster.credentials,
+      });
+      return WorkerState;
+    },
+  },
+
+  amiSet: {
```

Comment:
We have a convention of anything that's a `taskcluster-base.Entity` using class casing, so please call this AmiSet here, and the other places that you are referring to the thing returned by `AmiSet.setup` instead of an instance of an `AmiSet`.